Firebase/Messaging/FIRMessagingPendingTopicsList.m

```diff
@@ -160,9 +160,9 @@ - (void)addOperationForTopic:(NSString *)topic
     if (completion) {
       NSMutableArray *handlers = lastBatch.topicHandlers[topic];
```

Comment:
At this point the handlers variable is unused except for a nil check, and can probably be removed.

Reply:
We'd still need to reference the `lastBatch.topicHandlers[topic]` three times though, would it make sense to do this instead?:

```
NSMutableArray *handlers = lastBatch.topicHandlers[topic];
if (!handlers) {
  handlers = [NSMutableArray arrayWithCapacity:1];
}
[handlers addObject:completion];
lastBatch.topicHandlers[topic] = handlers; // Only new line.
```

Reply:
Done.